magento/magento2
# Fix #27985 CMS page link active on current page

### Description (*)
This PR helps to address issue #27985 , Added additional condition to check current page URL & link URL are equal.

### Fixed Issues (if relevant)

1. magento/magento2#27985: Privacy and Cookie Policy link not display as active on current page

### Manual testing scenarios (*)
1. Click "privacy and cookie policy" link in footer -> After opening the page "privacy and cookie policy" should be active.
2. Enter "baseurl/privacy-policy-cookie-restriction-mode/?test" -> After opening the page "privacy and cookie policy" should be active.
3. vendor/bin/phpunit -c dev/tests/unit/phpunit.xml.dist lib/internal/Magento/Framework/View/Test/Unit/Element/Html/Link/CurrentTest.php

### Questions or comments
If any please let me know your feedbacks.

### Contribution checklist (*)
